src/helpers.js

```diff
@@ -155,28 +155,22 @@ const generateInvalidTrace = async (
     JSON.stringify(await getDebugInfo(worker), null, 2),
     '```'
   ].join('\n'))
+
+  const location = {
+    file: filePath,
+    position: generateRange(textEditor, 0),
+  }
   const newIssueURL = `${issueURL}?title=${title}&body=${body}`
+
   return {
-    type: 'Error',
     severity: 'error',
-    html: `${escapeHTML(titleText)}. See the trace for details. ` +
-      `<a href="${newIssueURL}">Report this!</a>`,
-    filePath,
-    range: generateRange(textEditor, 0),
-    trace: [
-      {
-        type: 'Trace',
-        text: `Original message: ${ruleId} - ${message}`,
-        filePath,
-        severity: 'info',
-      },
-      {
-        type: 'Trace',
-        text: rangeText,
-        filePath,
-        severity: 'info',
-      },
-    ]
+    excerpt: `${escapeHTML(titleText)}. See the trace for details. ` +
+      `[Report this!](${newIssueURL})`,
```

Comment:
`newIssueURL` should be set as the `url` attribute. The message should be updated to instruct the user to click that.